src/main/resources/application.properties

```diff
@@ -16,3 +14,6 @@ jwt.token.prefix=Bearer
 # spring.flyway.password=
 ## Configure cors origins: must be a comma separated list without spaces ##
 # allowed.origins=http://127.0.0.1:3000,http://127.0.0.1:3001
+## Configure JWT
+jwt.signing.key=CHANGEME
+jwt.token.validity=1000
```

Comment:
Without this the maven build fails as the validity must be a number. Not sure if it is fine like this...

Reply:
works for me, I've updated it already in our cloud deployment